src/main/java/uk/gov/hmcts/document/DocumentClientApi.java

```diff
@@ -1,23 +1,52 @@
 package uk.gov.hmcts.document;
 
+import feign.Logger;
+import feign.codec.Encoder;
 import org.springframework.cloud.netflix.feign.FeignClient;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import org.springframework.context.annotation.Scope;
+import org.springframework.core.io.Resource;
 import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.util.MultiValueMap;
+import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.multipart.MultipartFile;
+import org.springframework.web.bind.annotation.RequestPart;
+import uk.gov.hmcts.document.domain.MultiValueMapEncoder;
 import uk.gov.hmcts.document.domain.UploadResponse;
 
-@FeignClient(name = "document.management.gateway.api", url = "${document.management.gateway.api.url}")
+@FeignClient(name = "document-management-gateway-api", url = "${document_management.api_gateway.url}",
+    configuration = DocumentClientApi.MultiPartSupportConfig.class)
 public interface DocumentClientApi {
 
-    @RequestMapping(method = RequestMethod.POST, value = "/documents",
-        headers = {"Content-Type: multipart/form-data",
-            "accept: application/vnd.uk.gov.hmcts.dm.document-collection.v1+hal+json;charset=UTF-8"
+    @RequestMapping(method = RequestMethod.POST, value = "/documents", consumes = "multipart/form-data",
```

Comment:
consumers should also use the constant `MediaType.MULTIPART_FORM_DATA_VALUE`

Reply:
done